storm-client/test/resources/ssl/README.md

```diff
@@ -0,0 +1,10 @@
+SSL test data
```

Comment:
How are the certifcates generated? How long are they valid? (Would be beneficial to document it
Would it be possible to generate them on the fly (i.e. via Maven) and create temporary keystores?

Reply:
The certificates are valid for 1000 years.  
```
~ % keytool -list -v -keystore testKeyStore.jks -storepass <> | grep valid
Valid from: Fri Jan 30 10:57:30 PST 2015 until: Tue Feb 07 10:57:30 PST 3015

~ % keytool -list -v -keystore testTrustStore.jks -storepass <> | grep Valid
Valid from: Fri Jan 30 10:57:30 PST 2015 until: Tue Feb 07 10:57:30 PST 3015
~ %
```